docs/_edot-sdks/java/troubleshooting.md

```diff
@@ -7,4 +7,57 @@ parent: EDOT Java
 
 # Troubleshooting the EDOT Java Agent
 
-TODO
\ No newline at end of file
+The sections below are in the order you should follow, unless you have already identified the section you need.
+
+This guide assumes you have tested the other components in the route from application+agent to Elastic Observability (eg collector or APM server, Elasticsearch, and Kibana) and that the problem has been isolated to the application+agent.
+
+## General
+
+Ensure you have set a service name (eg `-Dotel.service.name=Service1` or environment variable `OTEL_SERVICE_NAME` set to `Service1`) otherwise by default the data (traces, metrics, logs) will be sent to `unknown_service_java` - you may be getting data but it may all be under that service
+
+## Connectivity
+
+### To endpoint
+
+1. Check _from_ the host/VM/pod/container/image running the app, that connectivity is available to the APM server or collector. The examples here use a default URL, which you should replace with the endpoint you are using:
+
+- OpenTelemetry or EDOT collector: `curl -i http://127.0.0.1:4318/v1/traces -X POST -d '{}' -H content-type:application/json`
+- APM server on http: `curl --verbose -X GET http://127.0.0.1:8200`
+- APM server on https with secret token: `curl -X POST http://127.0.0.1:8200/ -H "Authorization: Bearer <secret_token>"`
+- APM server on https with API key: `curl -X POST http://127.0.0.1:8200/ -H "Authorization: ApiKey <api_key>"`
```

Comment:
```suggestion
- OpenTelemetry or EDOT collector without authentication: `curl -i http://127.0.0.1:4318/v1/traces -X POST -d '{}' -H content-type:application/json`
- APM server without authentication: `curl --verbose -X GET http://127.0.0.1:8200`
- APM server with secret token authentication: `curl -X POST http://127.0.0.1:8200/ -H "Authorization: Bearer <secret_token>"`
- APM server with API key authentication: `curl -X POST http://127.0.0.1:8200/ -H "Authorization: ApiKey <api_key>"`
```

Here the http/https part depends on how the collector is accessed in the URL, but in fact the only thing we care here is if there is authentication or not.